Dockerfile

```diff
@@ -1,7 +1,19 @@
-FROM alpine:3.10
+#############      builder                                  #############
+FROM golang:1.13.1 AS builder
 
-RUN apk add --update bash curl tzdata
+WORKDIR /go/src/github.com/gardener/machine-controller-manager
+COPY . .
+
+RUN .ci/build
+
+#############      base                                     #############
+FROM alpine:3.10 as base
 
-COPY bin/rel/machine-controller-manager /machine-controller-manager
+RUN apk add --update bash curl tzdata
 WORKDIR /
+
+#############      gardener-extension-hyper                 #############
```

Comment:
This should be "machine-controller-manager", right?

Reply:
Right :)

Reply:
Should be fixed now.